shared/deployments/staging.md

```diff
@@ -22,7 +22,7 @@
 ℹ Transaction hash: 0x0b4bef766abb49543810a242902fe9cea3d534184f8001ade47f341559982c2b
 ```
 
-- command: `./node_modules/.bin/lerna exec --scope="@tps/apps-address-book" --stream aragon apm publish major -- --files dist/ --environment staging`
+- command: `./node_modules/.bin/lerna exec --scope="@autarklabs/apps-address-book" --stream aragon apm publish major -- --files dist/ --environment staging`
```

Comment:
same here and for the rest of replacements in this file, it is documenting deployments that already happened